django_rq/utils.py

```diff
@@ -64,5 +64,10 @@ def get_statistics():
             queue_data['started_jobs'] = len(started_job_registry)
             queue_data['deferred_jobs'] = len(deferred_job_registry)
 
+            if RQ_SCHEDULER_INSTALLED:
+                scheduler = get_scheduler(queue.name)
+                queue_data['scheduled_jobs'] = len(
```

Comment:
This part fetches all scheduled jobs and compares `job.origin` with `queue.name`, which could be very expensive to run (and repeats it for each queue). Let's just add a `Scheduled Jobs` row for each connection and display the number of jobs in each scheduler, similar to `FailedQueue`.

This is a stop gap measure until we build out the proper infrastructure so that scheduler can keep track of the number of scheduled jobs on a per queue basis.